src/components/TitleLogo/styles.scss

```diff
@@ -0,0 +1,20 @@
+.title {
+  max-width: 500px;
```

Comment:
should we make the page dynamic for window resizing?

Reply:
We dont want the title to grow too large in desktop for prople with big screens. So the `max-width` property tells will set the maximum width to 500px. 

Devices small than that (mobile devices) will be dynamic